System.CommandLine.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27428.2037
```

Comment:
Please revert this.

Reply:
People using different versions of Visual Studio will cause this to change. It's just noise in the diff.

Reply:
Right. Completely agree on it going back. Just trying to figure out why it changed/how to avoid in the future.